src/smart-components/role/add-role-new/add-permissions.js

```diff
@@ -20,45 +20,19 @@ const selector = ({
   },
   roleReducer: { isRecordLoading, selectedRole },
 }) => ({
-  access: permission.data,
+  permissions: permission.data.map((p) => ({
+    application: p.application,
+    resource: p.resource_type,
+    operation: p.verb,
+    uuid: p.permission,
+  })),
```

Comment:
Can you style it a bit better?
```suggestion
  permissions: permission.data.map(({ application, resource_type: resource, verb, permission } = {}) => ({
    application,
    resource,
    operation: verb,
    uuid: permission,
  })),
```